serversession-backend-redis/src/Web/ServerSession/Backend/Redis/Internal.hs

```diff
@@ -282,14 +287,15 @@ insertSessionImpl session = do
       transaction $ do
         let sk = rSessionKey sid
         r <- batched (R.hmset sk) (printSession session)
-        -- TODO: R.expireat
+        expireSession session sto
+        -- Set the expiration if applicable
```

Comment:
Comment should be above.  But I think it's better to just remove the comment, `expireSession` is clear enough.